index.html

```diff
@@ -405,39 +515,75 @@ <h3 id="placement-of-mono-ruby">Placement of mono-ruby</h3>
 <section>
 <h3 id="placement-of-group-ruby">Placement of group-ruby</h3>
 
-<p>Group-ruby is placed as follows:</p>
+<p>
+<span class="checkme">
+In this section, placement rules of group-ruby are described as combinations 
+of two groups of characters, 
+one as 'Western characters' which has proportional width 
+and consisted with characters like 
+<a href="https://www.w3.org/TR/jlreq/#cl-24">Grouped numerals (cl-24)</a>,
+<a href="https://www.w3.org/TR/jlreq/#cl-25">Unit symbols (cl-25)</a>,
+<a href="https://www.w3.org/TR/jlreq/#cl-26">Western word space (cl-26)</a>,
+and 
+<a href="https://www.w3.org/TR/jlreq/#cl-27">Western characters (cl-27)</a>
+[[JLREQ]], 
+and another as 'Kana characters' which consisted with characters like 
+<a href="https://www.w3.org/TR/jlreq/#cl-15">Hiragana (cl-15)</a>,
+<a href="https://www.w3.org/TR/jlreq/#cl-26">Katakana (cl-16)</a>, and 
+<a href="https://www.w3.org/TR/jlreq/#cl-19">Ideographic characters (cl-19)</a>
```

Comment:
If "Kana characters" include other Ideographic characters, will it cause ambiguity?

Reply:
Ah, yes. Actually I wondered there is any good term for this,, in Japanese it's "Kana characters etc." or something actually. "Wide characters" might also be confusing,,. 
/cc @kidayasuo for additional help...

Reply:
I actually felt the original Japanese is also a bit misleading. As the original text is contrasting it with proportional characters, how about "fullwidth characters"?  The fullwidth-ness is used for controlling spacing in other places also (e.g. segment break transformation rules), the concept will be easily understood.
Another option I could think of is "Japanese characters" but this inherently ambiguous and also would trigger questions such as how about Chinese/Arabic/Hebrew characters?

Reply:
I agree that original text aimed to separate between Japanese full-width characters and half-width or proportional characters like US-Ascii or others,,, but I actually wondered the gap between Kana+ and wide or fullwidth characters, number of which might not be so large.
Considering background of this note, that focusing on Japanese ruby handling, I'd like "Japanese characters" not to have a (large) room of ambiguity

Reply:
I understand your reasoning and I am fine with "Japanese characters". Still, I feel it is more clear if we explain that it is fullwidth. proportional "Western" and fullwidth "Japanese".

Reply:
updated sections 2.3 and 3.5 where use "Kana characters"